diff --git a/node_modules/@vue/runtime-core/dist/runtime-core.cjs.js b/node_modules/@vue/runtime-core/dist/runtime-core.cjs.js
index 5ede0d7..91d742d 100644
--- a/node_modules/@vue/runtime-core/dist/runtime-core.cjs.js
+++ b/node_modules/@vue/runtime-core/dist/runtime-core.cjs.js
@@ -3737,7 +3737,7 @@ isSSR = false) {
     }
     if (isStateful) {
         // stateful
-        instance.props = isSSR ? props : reactivity.shallowReactive(props);
+        instance.props = isSSR ? props : reactivity.shallowReactive({...props, ...attrs});
     }
     else {
         if (!instance.type.props) {
diff --git a/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js b/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
index 41a5804..7765825 100644
--- a/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
+++ b/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
@@ -3755,7 +3755,7 @@ isSSR = false) {
     }
     if (isStateful) {
         // stateful
-        instance.props = isSSR ? props : shallowReactive(props);
+        instance.props = isSSR ? props : shallowReactive({...props, ...attrs});
     }
     else {
         if (!instance.type.props) {
